usecases/config/environment.go

```diff
@@ -218,6 +219,16 @@ func FromEnv(config *Config) error {
 		}
 	}
 
+	if v := os.Getenv("MODULES_CLIENT_TIMEOUT"); v != "" {
+		timeout, err := time.ParseDuration(v)
+		if err != nil {
+			return errors.Wrapf(err, "parse MODULES_CLIENT_TIMEOUT as time.Duration")
+		}
+		config.ModuleHttpClientTimeout = timeout
+	} else {
+		config.ModuleHttpClientTimeout = 60 * time.Second
```

Comment:
Can we change the default timeout to 50 seconds?

Reply:
Sure, changed it now.